front-end/src/components/Table/RankingTable/OverallTable.js

```diff
@@ -154,19 +213,21 @@ class OverallTable extends React.Component {
             </TableRow>
           </TableHead>
           <TableBody>
-            {this.chooseTableRow("1", classes)}
-            {this.chooseTableRow("2", classes)}
-            {this.chooseTableRow("3", classes)}
-            {this.chooseTableRow("4", classes)}
-            {this.chooseTableRow("5", classes)}
-            {this.chooseTableRow("6", classes)}
-            {this.chooseTableRow("7", classes)}
-            {this.chooseTableRow("8", classes)}
+            {this.chooseTableRow(1, classes)}
```

Comment:
Anh vẫn chưa dùng loop giống em nói. 
this.state.overallRanking.map(user => (
      this.chooseTableRow(...)
));
Kiểu thế. Anh viết code gọn nhất có thể nha là được.